docs/source/getting-started/stats/index.rst

```diff
@@ -0,0 +1,10 @@
+Statistical Modelling
+=====================
+
+Beyond simple means and medians, a range of statistical techniques have DP variants.
+Some, like linear regression, can be built with the tools we've already introduced.
+Others, like PCA, are supported under the ``opendp.sklearn`` namespace.
```

Comment:
I would say that we intend to provide more functionality in this space in the future and are accepting contributions. For the time being, we have PCA and consider it a model for future functionality.